pkg/oc/bootstrap/docker/up.go

```diff
@@ -891,30 +885,31 @@ func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
 }
 
 // CheckAvailablePorts ensures that ports used by OpenShift are available on the Docker host
-func (c *CommonStartConfig) CheckAvailablePorts(out io.Writer) error {
+func (c *CommonStartConfig) CheckAvailablePorts(out io.Writer) (bool, error) {
 	if !c.checkAlternatePorts {
 		err := c.OpenShiftHelper().TestPorts(openshift.DefaultPorts)
 		if err == nil {
-			return nil
+			return false, nil
 		}
-		return errors.NewError("a port needed by OpenShift is not available").WithCause(err)
+		return false, errors.NewError("a port needed by OpenShift is not available").WithCause(err)
 	}
 	err := c.OpenShiftHelper().TestPorts(openshift.AllPorts)
 	if err == nil {
-		return nil
+		return false, nil
 	}
 	if !openshift.IsPortsNotAvailableErr(err) {
-		return err
+		return false, err
 	}
 	unavailable := sets.NewInt(openshift.UnavailablePorts(err)...)
 	if unavailable.HasAny(openshift.BasePorts...) {
-		return errors.NewError("a port needed by OpenShift is not available").WithCause(err)
+		return false, errors.NewError("a port needed by OpenShift is not available").WithCause(err)
 	}
+	useAlternateDNSPort := false
```

Comment:
Why do we have this lever?  can't we just always use the alternate port and never try 53?

Reply:
yes, we probably could